README.Rmd

```diff
@@ -0,0 +1,31 @@
+---
+output: github_document
+bibliography: vignettes/references.bib
+link-citations: true
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+[![Travis-CI Build Status](https://travis-ci.org/sbfnk/bpmodels.svg?branch=master)](https://travis-ci.org/sbfnk/bpmodels)
+[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/y37i8x0wo9o8s2wf?svg=true)](https://ci.appveyor.com/project/sbfnk/bpmodels)
+[![codecov](https://codecov.io/github/sbfnk/bpmodels/branch/master/graphs/badge.svg)](https://codecov.io/github/sbfnk/bpmodels) 
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+`bpmodels` is an R package for simulating epidemics using branching processes. The package provides functions for fitting the following models:
```

Comment:
Is there something missing here (which models)? Also at the moment we don't have any functions for fitting, right? Just simulating and calculating likelihoods.

Reply:
That has been resolved.